.env.template

```diff
@@ -161,8 +141,48 @@
 # AAVE_FUNDER=0xbe0eb53f46cd790cd13851d5eff43d12404d33e8
 # IDLE_FUNDER=0x2bc44f6c34d9200c258ac07a56d81963d4f92c4c
 # SUSHI_FUNDER=0xf977814e90da44bfa03b6295a0616a897441acec
-# LQTY_FUNDER=0xa9fabaf4a8d01177458e68b8419d138b50ff8ca7
 # DODO_FUNDER=0xf977814e90da44bfa03b6295a0616a897441acec
+# BOND_FUNDER=0x0F4ee9631f4be0a63756515141281A3E2B293Bbe
+
+# ALPHAHOMORA_IBUSDT=0x020eDC614187F9937A1EfEeE007656C6356Fb13A
+# ALPHAHOMORA_IBUSDC=0x08bd64bfc832f1c2b3e07e634934453ba7fa2db2
+# ALPHAHOMORA_IBETH=0xeea3311250fe4c3268f8e684f7c87a82ff183ec1
+
+# CONVEX_BOOSTER=0xF403C135812408BFbE8713b5A23a04b3D48AAE31
+# CONVEX_REWARDS=0x689440f2ff927e1f24c72f1087e1faf471ece1c8
+
+# GUNI_ROUTER=0x8CA6fa325bc32f86a12cC4964Edf1f71655007A7
+# GUNI_RESOLVER=0x3B01f3534c9505fE8e7cf42794a545A0d2ede976
+# GUNI_POOL_USDC_USDT=0x02f88164060912ee44ba9480d05d462d20cff3bc
+
+# DODO=0x43Dfc4159D86F3A37A5A4B3D4580b888ad7d4DDd
+# DODO_PROXY=0xa356867fDCEa8e71AEaF87805808803806231FdC
+# DODO_MINE=0xaeD7384F03844Af886b830862FF0a7AFce0a632C
+# DODO_APPROVE=0xCB859eA579b28e02B87A1FDE08d087ab9dbE5149
+# DODO_V1_DODO_USDT=0x8876819535b48b551C9e97EBc07332C7482b4b2d
+
+# BARNBRIDGE_CUSDC=0x4B8d90D68F26DEF303Dcb6CFc9b63A1aAEC15840
```

Comment:
Please revert the non-BarnBridge changes in this file. Only add the BARNBRIDGE_* envs.